src/commonMain/kotlin/exchange.dydx.abacus/functional/vault/VaultAccount.kt

```diff
@@ -68,16 +79,16 @@ object VaultAccountCalculator {
         vaultInfo: AccountVaultResponse,
         vaultTransfers: IndexerTransferBetweenResponse
     ): VaultAccount {
-        val presentValue = vaultInfo.equity
+        val presentValue = vaultInfo.equity?.let { it / 1_000_000 }
```

Comment:
do we have a standard value const for `1_000_000`?

Reply:
I don't think so, I'll add one in a follow up